src/python_testing/TC_OCC_3_1.py

```diff
@@ -29,19 +29,29 @@
 
 import logging
 import time
+from typing import Optional, Any
 
 import chip.clusters as Clusters
-from chip import ChipDeviceCtrl
 from chip.interaction_model import Status
 from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
 from mobly import asserts
 
 
 class TC_OCC_3_1(MatterBaseTest):
-    async def read_occ_attribute_expect_success(self, endpoint, attribute):
+    async def read_occ_attribute_expect_success(self, attribute):

```

Comment:
Not a problem to expose explicitly, but read_single_attribute_check_success defaults to the matter_test_config.endpoint

Reply:
Will do as follow-up.